app/javascript/readingList/components/TagList.jsx

```diff
@@ -0,0 +1,88 @@
+import { h } from 'preact';
+import PropTypes from 'prop-types';
+
+function LargeScreenTagList({ availableTags, selectedTag, onSelectTag }) {
+  return (
+    <fieldset className="hidden grid grid-cols-1 gap-2">
+      <legend className="hidden">Filter by tag</legend>
+      {availableTags.map((tag) => (
+        <label
+          className={`crayons-link crayons-link--block${
+            selectedTag === tag ? ' crayons-link--current' : ''
+          }`}
+          aria-label={`${tag} tag`}
+        >
+          <input
+            type="radio"
+            name="filterTag"
+            onClick={onSelectTag}
+            key={tag}
+            className="opacity-0"
+            checked={selectedTag === tag}
+            value={tag}
+          />
+          #{tag}
+        </label>
+      ))}
+    </fieldset>
+  );
+}
+
+/**
+ *
+ * @param {object} props
+ * @param {Array<string>} props.availableTags A list of available tags.
+ * @param {string} [props.selectedTag=''] The currently selected tag.
+ * @param {function} A handler for when the selected tag changes.
```

Comment:
```suggestion
 * @param {function} props.onSelectTag A handler for when the selected tag changes.
```
Also we should add the `isMobile` param